app/data/domain/cdoTaskList.js

```diff
@@ -165,14 +167,61 @@ class CdoTaskList {
     )
   }
 
+  get neuteringRulesPassed () {
+    if (!CdoTaskList.dateStageComplete(this._cdo.exemption.verificationDatesRecorded)) {
+      return false
+    }
+
+    if (this._cdo.dog.breed !== 'XL Bully') {
+      return false
+    }
+
+    // Date of Birth must be less than 16 months ago
+    if (!this._cdo.dog.youngerThanSixteenMonths) {
+      return false
+    }
+
+    const endOfDay = new Date()
+    endOfDay.setHours(23, 59, 59, 999)
+
+    // Neutering deadline today
+    return CdoTaskList.dateStageComplete(this._cdo.exemption.neuteringDeadline) &&
+        this._cdo.exemption.neuteringDeadline >= endOfDay
+  }
+
+  get microchipRulesPassed () {
+    if (!CdoTaskList.dateStageComplete(this._cdo.exemption.verificationDatesRecorded)) {
+      return false
+    }
+
+    if (!CdoTaskList.dateStageComplete(this._cdo.exemption.microchipDeadline)) {
+      return false
+    }
+
+    // Microchip deadline yesterday
+    return Date.now() < this._cdo.exemption.microchipDeadline.getTime()
```

Comment:
Why the different logic between this and line 189? I think they are both correct but using different styles and subjects are switched around

Reply:
have written a test to check this...both microchip deadline and neutering deadline will be set at 00:00, on that specific day - so if it's 1am BST it will no longer be completable